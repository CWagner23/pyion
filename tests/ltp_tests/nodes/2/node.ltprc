1 300

# Connection to node 1
a span 1 2 2 1200 100100 1 'udplso 196.128.0.101:3001 230000'

# Loopback
a span 2 2 2 1200 100100 1 'udplso 196.128.0.102:3001 230000'

w 1

# One input for all incoming connections
s 'udplsi 196.128.0.102:3001'
