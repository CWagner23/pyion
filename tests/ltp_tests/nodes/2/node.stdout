[i] admin pgm using SDR parm overrides from node.ionconfig.
wmKey:           0
wmSize:          5000000
wmAddress:       0
sdrName:        ''
sdrWmSize:       0
configFlags:     1
heapWords:       5000000
heapKey:         -1
logSize:         0
logKey:          -1
pathName:       '/usr/ion'
[?] No such directory; disabling heap residence in file and transaction reversibility: /usr/ion
Stopping ionadmin.
Stopping ionsecadmin.
Stopping ionadmin.
Stopping ltpadmin.
Stopping ipnadmin.
Stopping bpadmin.
sssNOTE: udplso now gets transmission data rate from the contact plan.  txbps is still accepted on the command line, for backward compatibility, but it is ignored.
gtssssgts